images/openshift-enterprise-pod.yml

```diff
@@ -1,15 +1,11 @@
-container_yaml:
-  go:
-    modules:
-    - module: github.com/openshift/images/pod
-      path: pod
 content:
   source:
-    dockerfile: pod/Dockerfile.rhel
+    dockerfile: Dockerfile.Rhel
+    path: build/pause
```

Comment:
We're just about there.  Minor nit to avoid unnecessary churn in the future: fields are alphabetized, so path should go after (but not under!) git.

Reply:
Fixed :)